core/test/integration/api/advanced_browse_spec.js

```diff
@@ -126,10 +126,10 @@ describe('Filter Param Spec', function () {
             });
         });
 
-        describe('4. Posts - filter="author:[leslie,pat]+(featured:true,tag:audio)"', function () {
+        describe('4. Posts - filter="author:[leslie,pat]+(tag:audio,image:-null)"', function () {
```

Comment:
This test was changed to use more complex not-null logic in order to show the bug that was fixed in gql@0.0.3